cmd/infracost/run.go

```diff
@@ -264,6 +265,37 @@ func runMain(cmd *cobra.Command, runCtx *config.RunContext) error {
 	return nil
 }
 
+func buildPriorProjects(snapshot string) (map[string]*schema.Project, error) {
+	priorProjects := make(map[string]*schema.Project)
+	prior, err := loadInfracostJSONSnapshot(snapshot)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, p := range prior.Projects {
+		priorProjects[p.Name] = p.ToSchemaProject()
+	}
+
+	return priorProjects, nil
+}
+
+func loadInfracostJSONSnapshot(snapshot string) (output.Root, error) {
+	_, err := os.Stat(snapshot)
+	if err != nil {
+		return output.Root{}, fmt.Errorf("%s used by --compare-to flag does not exist", snapshot)
+	}
+
+	b, _ := os.ReadFile(snapshot)
```

Comment:
Is it okay to skip this error?

Reply:
resolved in https://github.com/infracost/infracost/pull/1570/commits/73e7cf37e959cbac828e2e5ad4d88cbf4c59a559